docs/edd-request-form.html

```diff
@@ -141,23 +141,27 @@
         </div>
         <div class="nypl-row">
         <div class="nypl-column-one-quarter">
-          <div class="nypl-text-field" id="article-title">
-            <label for="required-field" id="article-title-label">Starting Page:
+          <div class="nypl-text-field" id="article-start-page">
+            <label for="required-start-page-field" id="article-start-page-label">Starting Page:
               <span class="nypl-required-field">Required</span>
             </label>
-            <input id="required-field" type="text" required="true" aria-labelledby="article-title-label article-title-status" aria-required="true" placeholder="" />
-            <!-- <span class="nypl-field-status" id="article-title-status" aria-live="assertive" aria-atomic="true">Please indicate the title of the chapter or article </span> -->
+
+            <input id="required-start-page-field" type="text" required="true" aria-labelledby="article-start-page-label article-start-page-status" aria-required="true" placeholder="" />
+
+            <span class="nypl-field-status" id="article-start-page-status" aria-live="assertive" aria-atomic="true">Values must be numeric only (No special characters allowed)</span>
           </div>
-          <div class="nypl-text-field" id="article-title">
-            <label for="required-field" id="article-title-label">Ending Page:
+          <div class="nypl-text-field" id="article-end-page">
+            <label for="required-end-page-field" id="article-end-page-label">Ending Page:
               <span class="nypl-required-field">Required</span>
             </label>
-            <input id="required-field" type="text" required="true" aria-labelledby="article-title-label article-title-status" aria-required="true" placeholder="" />
-            <span class="nypl-field-status" id="article-title-status" aria-live="assertive" aria-atomic="true">Values must be numeric only (No special characters allowed)</span>
```

Comment:
They could be special characters (or nothing). I think a bigger problem will be people who don't have the page numbers ("I only know it's chapter 7"). Is there anything we can say for that situation ("Just enter anything and explain it in the note") or is it best to say nothing?